packages/salesforce-adapter/src/filters/custom_objects_from_soap_describe.ts

```diff
@@ -88,19 +91,19 @@ const addSObjectInformationToInstance = async (
     return compoundName
   }
 
-  // Only fields with "child's" referring to a field as it's compoundField
+  // Only fields that have a child referring to a field as its compoundField
```

Comment:
thanks for the english fix, I think we can use a slightly different terminology to be more consistent with the code maybe?
```suggestion
  // Only fields that have another field referring to them as a compoundField
```